source/grid/tria.cc

```diff
@@ -1825,9 +1855,10 @@ namespace internal
           // nodes (for example a
           // ring of cells -- no end
           // points at all)
-          AssertThrow(((spacedim == 1) && (boundary_nodes == 2)) ||
-                        (spacedim > 1),
-                      ExcMessage("The Triangulation has too many end points"));
+          // AssertThrow(((spacedim == 1) && (boundary_nodes == 2)) ||
+          //              (spacedim > 1),
+          //            ExcMessage("The Triangulation has too many end
+          //            points"));
```

Comment:
Can you explain why it is removed? I suppose this will get enabled again at some point?